src/ts/background/index.ts

```diff
@@ -1,11 +1,10 @@
-import {createStore, Store} from 'redux';
-import {wrapStore} from 'webext-redux';
-import reducers, {IAppState} from './store';
-import {getStateFromStorage, saveStateToStorage} from '../utils/storage';
-
-(async () => {
-    const store: Store<IAppState> = createStore(reducers, await getStateFromStorage());
+import {createStore, Store} from 'redux'
+import {wrapStore} from 'webext-redux'
+import reducers, {IAppState} from './store'
+import {getStateFromStorage, saveStateToStorage} from '../utils/storage'
+;(async () => {
```

Comment:
🤔

Reply:
more fighting with prettier :)
(this is an issue with `semi: false`)

see: https://github.com/prettier/prettier/issues/2800

Reply:
>more fighting with prettier :)  
I reserve right to be unhappy with it's decisions 😛  this looks like a bug ;)

Reply:
@Stvad please read the discussion. See this comment: https://github.com/prettier/prettier/pull/1129#issuecomment-292243414 

Also this issue for more discussion on the same: https://github.com/prettier/prettier/issues/12

It would be an actual bug to **not** do this, & use autoformatting, as it can cause the JS to break;

We can either keep semicolons, or this. That's just how prettier works 🤷🏽‍♂️

Reply:
We have to do one of the two things -- as it is not technically a bug . Question is, what makes you less unhappy 🙂?

Reply:
let's stick with removing semicolons :)